pkg/antctl/raw/multicluster/get/resourceimport.go

```diff
@@ -134,24 +120,24 @@ func runE(cmd *cobra.Command, args []string) error {
 		if !unversioned && len(gvks) == 1 {
 			resourceImport.SetGroupVersionKind(gvks[0])
 		}
-		res = resourceImport
+		resImports = append(resImports, resourceImport)
 	} else {
 		resourceImportList := &multiclusterv1alpha1.ResourceImportList{}
 		err = k8sClient.List(context.TODO(), resourceImportList, &client.ListOptions{Namespace: options.namespace})
 		if err != nil {
 			return err
 		}
+		resImports = resourceImportList.Items
+	}
 
-		if len(resourceImportList.Items) == 0 {
-			if options.namespace != "" {
-				fmt.Fprintf(cmd.ErrOrStderr(), "No resources found in Namespace %s\n", options.namespace)
-			} else {
-				fmt.Fprintln(cmd.ErrOrStderr(), "No resources found in all Namespaces")
-			}
-			return nil
+	if len(resImports) == 0 {
+		if options.namespace != "" {
+			fmt.Fprintf(cmd.ErrOrStderr(), "No resources found in Namespace %s\n", options.namespace)
+		} else {
+			fmt.Fprintln(cmd.ErrOrStderr(), "No resources found in all Namespaces")
 		}
-		res = resourceImportList.Items
+		return nil
 	}
 
-	return output(res, singleResource, options.outputFormat, resourceimport.Transform)
+	return output(resImports, false, options.outputFormat, resourceimport.Transform)
```

Comment:
What will be changed after changing "singleResource" to "false"?

Reply:
There is actually no difference since both cases eventually call `objectTransform`. After replace the old  single `res` with `resImports`, it can always be `false`.

Reply:
It turns out there are slight differences between single and list. The output will be wrapped by `[]` if it's list. I will revert this change.

Reply:
Reverted